components/browser/state/src/main/java/mozilla/components/browser/state/state/CustomTabConfig.kt

```diff
@@ -26,8 +28,8 @@ import androidx.browser.customtabs.CustomTabsSessionToken
  * @property sessionToken The token associated with the custom tab.
  */
 data class CustomTabConfig(
-    val id: String,
```

Comment:
nit: Not 100% sold on the name since it differs from our other `*State` classes here. But that's something quick to change - especially when this is not used by apps as you said.